.github/workflows/ci.yml

```diff
@@ -11,6 +11,24 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
+
+      - name: Check if it is a dependency that should be managed by Expo
+        if: github.actor == 'dependabot[bot]'
```

Comment:
this shouldn't run if it's not a pull request, we need to put a condition in here